src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -912,8 +913,8 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
 
   public partitionsOf = async (params: PartitionsOfParams): Promise<Partition[]> => {
     const partitions = await (await this.contract).partitionsOf.callAsync(params.tokenHolder);
-    return bytes32ArrayToStringArray(partitions).map(element => {
-      return element === Partition.Unlocked ? Partition.Unlocked : Partition.Undefined;
+    return partitions.map(element => {
```

Comment:
This can be rewritten as `partitions.map(parsePartitionBytes32Value)`

Reply:
Thanks I will make the change ASAP